WPMedia/fusion-news-theme-blocks
# TMEDIA-363: Add Lead Art block to StoryBook

## Description
Add Lead Art block to StoryBook

## Jira Ticket
- [TMEDIA-363](https://arcpublishing.atlassian.net/browse/TMEDIA-363)

## Acceptance Criteria
Add the following blocks to the Blocks storybook, and set them up for visual regression testing with Chromatic: 
  * Lead Art
    
## Test Steps
1. Checkout this branch `git checkout TMEDIA-363-add-lead-art-to-storybook`
2. Run `npm run storybook` and check the Lead Art category in Blocks.

## Effect Of Changes
### Before
Nothing

### After
![image](https://user-images.githubusercontent.com/2287238/127572606-8ea13097-07de-471d-8e1d-a757c75d4398.png)

## Dependencies or Side Effects
N/A

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
